src/panoptes/pocs/filterwheel/filterwheel.py

```diff
@@ -212,6 +236,23 @@ def move_to(self, position, blocking=False):
 
         return self._move_event
 
+    def move_to_dark_position(self):
+        """ Move to filterwheel position for taking darks. """
+        if self.dark_position:
+            self.move_to(self.dark_position)
+        else:
+            msg = f"Request to move to dark position but {self} has no dark_position set."
+            self.logger.warning(msg)
+
+    def move_to_light_position(self):
+        """ Return to previous filterwheel position from before taking darks. """
+        if self._light_position:
+            self.move_to(self._light_position)
+        else:
+            msg = f"Request to revert to previous light position but {self} has" + \
+                "no _light_position stored."
```

Comment:
Let's change to not expose the private variable name in the log message, especially since the user doesn't have the option to change or set that in any way.

Reply:
Done.